docs/models.md

```diff
@@ -6,9 +6,9 @@ These are:
 * `Parameter`: A parameter represents some global variable in the tax-benefit system. For example, the personal allowance in the UK income tax system is a parameter.
 * `ParameterValue`: A parameter value represents a specific value for a parameter at a given range in time (e.g. the personal allowance between April 2023 and April 2024 is £12,570).
 * `Policy`: A policy represents a set of `ParameterValue`s that together define a specific change to the tax-benefit system. For example, a policy might represent a change to the personal allowance, or a change to the rate of VAT.
-* `Dynamics`: A dynamics represents a set of rules that define how the tax-benefit system changes over time. For example, a dynamics might represent the annual uprating of parameters in the tax-benefit system.
+* `Dynamic`: A dynamic represents a set of rules that define how the tax-benefit system changes over time. For example, a dynamic might represent the annual uprating of parameters in the tax-benefit system.
```

Comment:
Uprating as a dynamic? This is policy at least in the US. I think of dynamics as the feedbacks in the economy. We might want a separate term/concept for policy uncertainty.